autoload/lsp/ui/vim/signature_help.vim

```diff
@@ -51,6 +83,30 @@ function! s:handle_signature_help(server, data) abort
     endif
 endfunction
 
+function! s:get_parameter_label(signature, parameter) abort
+    if has_key(a:parameter, 'label')
+        if type(a:parameter) == v:t_list
+            let l:string_range = a:parameter['label']
+            return strcharpart(a:signature['label'], l:string_range[0], l:string_range[1])
```

Comment:
The last argument of `strcharpart` is a length, but LSP returns the exclusive end offset instead.

Reply:
Thanks!
I had two mistake... and resolved its.

I checked following code.

```VimL
" 012345
echo s:get_parameter_label({ 'label': '012345' }, { 'label': [0, 6] })

" 01234
echo s:get_parameter_label({ 'label': '012345' }, { 'label': [0, 5] })

" 12345
echo s:get_parameter_label({ 'label': '012345' }, { 'label': [1, 6] })

" 1234
echo s:get_parameter_label({ 'label': '012345' }, { 'label': [1, 5] })
```